requirements.txt

```diff
@@ -18,7 +18,7 @@ pyqtgraph
 matplotlib
 autodepgraph
 networkx
-qutechopenql
+qutechopenql>=0.10.3
```

Comment:
Wouldn't this requirement make it harder for those that still use OpenQL for CCL (requiring <0.10.0)?  Granted pip would always install latest version of requirements regardless, but after manually downgrading qutechopenql, this would make the requirements for PycQED fail upon re-installation

Reply:
true, although you would need to build OpenQL manually anyway. The reason to include this is to not be satisfied with 0.10.0, the latest available on PyPi (0.10.3 is only available on https://github.com/DiCarloLab-Delft/OpenQL/releases)